jest.config.js

```diff
@@ -16,7 +17,4 @@ module.exports = {
       },
     },
   },
-  moduleNameMapper: {
-    "package.json": "<rootDir>/tests/stubs/package.json",
-  },
```

Comment:
Was causing this error:

![image](https://user-images.githubusercontent.com/284476/62826223-cadc7200-bb85-11e9-8716-79aa90e3ea8c.png)

But oddly only when doing `jest --watch`. That is, config worked as is with `jest`. Anyways, appears unused.